src/swaps/liqualityboost/liqualityBoostERC20toNative/LiqualityBoostERC20toNative.ts

```diff
@@ -238,7 +238,7 @@ class LiqualityBoostERC20toNative extends SwapProvider {
       return {
         ...resultLSP,
         ...swapLSP,
-        toAmount: resultLSP.toAmount,
+        toAmount: (resultLSP as any).toAmount,
```

Comment:
why all the `as any` all of a sudden 😅

Reply:
Have to change it to `LiqualitySwapHistoryItem`